Ginger/GingerCoreNET/WorkSpaceLib/IWorkSpaceEventHandler.cs

```diff
@@ -38,5 +38,6 @@ public interface IWorkSpaceEventHandler
         void ShowUpgradeGinger(string solutionFolder, List<string> higherVersionFiles); // when openning solution we check if the solution is older or newer, and popup an upgrade question for ginger or for the solution
         void ShowUpgradeSolutionItems(SolutionUpgradePageViewMode upgradeSolution, string solutionFolder, string solutionName, List<string> list);
         void SetSolutionSourceControl(Solution solution, ref string RepositoryRootFolder);
+        bool ValidateEncryptionKey(Solution solution);
```

Comment:
This even name is confusing. Let's rename it to InputEncryptionKeyHandler or something like this

Reply:
Changed to OpenEncryptionKeyHandler